### ISSUE TITLE
w2ui form not working in IE 11

### ISSUE THREAD

**Issue by StalderT** [vitmalina/w2ui#2311]
Hello,

If I try this simple example in IE 11 with w2ui 1.5 : 

```
    $(function() {
        $('#myForm').w2form({
            name: 'myForm',
            fields: [{
                name: 'first_name',
                type: 'text',
                required: true
            }, {
                name: 'last_name',
                type: 'text',
                required: true
            }, {
                name: 'comments',
                type: 'text'
            }],
            actions: {
                Reset: function(){
                    this.clear();
                },
                Save: function(){
				    alert("Save");
                    this.save();
                }
            }
        });
    });
```

I have the error :
![image](https://user-images.githubusercontent.com/13799456/193587779-fa647d66-947a-45cd-bd96-141f83b48541.png)

Have you an idea how to fix this issue ?

Thanks in advance.

Regards.

**Comment by vitmalina** [comment]
Not sure. v 15 should work in IE11. What version of jQuery do you use. Can you try to upgrade it to latest or downgrade it. The error sims to suggest it might be related to jQuery version, also, it is in foreign language for me.

**Comment by StalderT** [comment]
Hello,

I am using jquery v 3.6.1

I tested with 2.2.4 and 1.12.4 and got the same error.

All other w2ui components work fine.

It would be great if this problem could be fixed.

**Comment by vitmalina** [comment]
This issue is now fixed in 1.5 branch. I have also recompiled updated version of w2ui in here https://raw.githubusercontent.com/vitmalina/w2ui/w2ui-1.5/dist/w2ui.js
